src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -348,6 +351,18 @@ else if (VERSION_CHECK_CAT.equals(event.getConfigID()))
         }
     }
 
+    @Subscribe
+    public void missingMapping(FMLMissingMappingsEvent event){
+        for (FMLMissingMappingsEvent.MissingMapping evt:event.getAll())
```

Comment:
Spacing
```java
for (FMLMissingMappingsEvent.MissingMapping evt : event.getAll())
```
The variable is better called `entry` instead of `evt` (event)